packages/ts/react-auth/src/useAuth.tsx

```diff
@@ -18,38 +18,20 @@ const LOGOUT = 'LOGOUT';
  * The user is not required to comply with this format. This is just for convenience.
  */
 export type AuthUser = Readonly<{
-  sub?: string;
-  name?: string;
-  givenName?: string;
-  familyName?: string;
-  middleName?: string;
-  nickName?: string;
-  preferredUsername?: string;
-  profile?: string;
-  picture?: string;
-  website?: string;
-  email?: string;
-  emailVerified?: boolean;
-  gender?: string;
-  birthdate?: string;
-  zoneinfo?: string;
-  locale?: string;
-  phoneNumber?: string;
-  phoneNumberVerified?: boolean;
-  roles?: string[];
+  roles: string[];
```

Comment:
Regarding roles: I just made them required for now, so you'd have to add them in order to use the package. I was thinking about adding an accessor function, but thought that might be more hassle than just having a `roles` property as convention.

Reply:
SSO Kit doesn't have roles right now. I'm not sure that imposing them is a good idea, given that we're giving freedom about the `User` object.
What about allowing to define a custom role function while giving a default one that tries to access `roles` as a `string[]`?